ts/components/conversation/composition/CompositionInput.tsx

```diff
@@ -112,6 +112,25 @@ function removeHtmlCharactersBeforeIndex(
   return { newHtml, newIndex };
 }
 
+/**
+ * Remove html at a html index.
+ * @param originalHtml - String to mutate
+ * @param htmlIdx - Html index to remove from (not including the character at this index). @see {@link getHtmlIndexFromSelection}
+ * @param numberOfCharactersToRemove - Number of characters to remove before the index.
+ */
+function removeHtmlCharactersAfterIndex(
+  originalHtml: string,
+  htmlIdx: number,
+  numberOfCharactersToRemove: number
+): { newHtml: string; newIndex: number } {
+  // TODO: handle when htmlIdx - toRemove < 0
+  const before = originalHtml.slice(0, htmlIdx);
+  const after = originalHtml.slice(htmlIdx + numberOfCharactersToRemove);
+  const newHtml = before + after;
+  const newIndex = htmlIdx;
+  return { newHtml, newIndex };
+}
```

Comment:
I had a quick look and I don't think merging those two functions make much sense. It will just make them both less readable, so I'd say keep them separate for now.

Should you fix those though?
`  // TODO: handle when htmlIdx - toRemove < 0`